src/clj/puppetlabs/services/protocols/filesystem_watch_service.clj

```diff
@@ -0,0 +1,38 @@
+(ns puppetlabs.services.protocols.filesystem-watch-service
```

Comment:
Same as a comment in the implementation, but other tk service protocols use the `puppetlabs.trapperkeeper.services.protocols.` namespace. Seems like consistency would be good there.